contracts/crowdsale/ROUX_RefferalBasedCrowdsale.sol

```diff
@@ -24,9 +24,12 @@ contract ROUX_ReferralBasedCrowdsale is MintedCrowdsale, PostDeliveryCrowdsale,
 
     mapping(address => Backer) mBackers;
 
-    uint256 mMaxReferralPercentageFee = 3;
-    uint256 mMaxDiscount = 20;
-    uint256 oneHundred = 100;
+    // Maximum allowable referral reward percentage (i.e. 300 = 3%)
+    uint256 mMaxReferralPercentageFee = 300;
+    // Maximum allowable token discount percentage (i.e. 2000 = 20%)
+    uint256 mMaxDiscount = 2000;
```

Comment:
Let's set the maximum to 10000 (100%) and the default to 2000.

Reply:
souds good

Reply:
dont you think we should have some of maximum that serves as a safe gaurd?

Reply:
I would rather not give it any default